assets/src/edit-story/components/panels/design/layer/layerPanel.js

```diff
@@ -27,31 +28,56 @@ import { LAYER_HEIGHT, DEFAULT_LAYERS_VISIBLE } from './constants';
 import LayerList from './layerList';
 import useLayers from './useLayers';
 
+const Container = styled.div`
+  margin: 0 4px;
+  filter: drop-shadow(0px -4px 8px rgba(0, 0, 0, 0.2));
```

Comment:
Does this duplicate? Would it be good to store in the theme? Or at least pull the rgba in as a theme color added?

Reply:
Looks like this same value is used here: https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/elements/video/controls.js#L85-L91

This was an old change so I don't know if it will survive. I don't see the color anywhere else, so I'd say we leave it for now 🤷

Reply:
works for me!